K2Bridge/Visitors/MetricAggregationVisitor.cs

```diff
@@ -57,5 +59,21 @@ public void Visit(SumAggregation sumAggregation)
 
             sumAggregation.KustoQL = $"['{sumAggregation.Key}']={KustoQLOperators.Sum}({sumAggregation.Field})";
         }
+
+        /// <inheritdoc/>
+        public void Visit(PercentileAggregation percentileAggregation)
+        {
+            Ensure.IsNotNull(percentileAggregation, nameof(percentileAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(percentileAggregation.Field, percentileAggregation.Field, ExceptionMessage);
+
+            /// Median is Percentile(fieldName, 50)
+            percentileAggregation.KustoQL = string.Empty;
+
+            var valuesForColumnNames = string.Join('%', percentileAggregation.Percents.ToList().Select(item => $"{item:0.0}"));
+            var valuesForOperator = string.Join(',', percentileAggregation.Percents);
+
+            // Example: ['A%percentile%25.0%50.0%99.0%False']=percentiles_array(fieldA, 25,50,99)']
```

Comment:
looks much better.
to me the 'keyed' true/false. which means median or percentile is not self explanatory.

worth adding yet another  comment here on this

Reply:
done

Reply:
talked f2f